### DISCUSSION TITLE
Imagesnap problem

### DISCUSSION THREAD

**Discussion by arijitdas123student** [just4give/balena-edge-impulse-linux#7] [General]
Hey @just4give as per I have checked on Github and moreover on Stackoverflow I get to see that there is a way by which we can do the installation on aarch64.
![Screenshot (123)](https://user-images.githubusercontent.com/64097541/115555632-5865e400-a2cd-11eb-8127-bd28cb3dad58.png)
Check this out and lemme know if that works out ?! ⬆️

**Comment by just4give** [comment]
Thanks Arijit but no luck. I am guessing some balena settings but don't know what ....

<img width="1185" alt="Screen Shot 2021-04-21 at 1 17 47 PM" src="https://user-images.githubusercontent.com/9275193/115638254-07132000-a2e0-11eb-85af-2e6a8d922b44.png">

**Reply by arijitdas123student** [reply]
Hmm the website DID say that it works on ARM32/aarch64 but I get to see the error that it requires x86 only! :(
